### PR TITLE
{bio}[GCCcore/11.2.0] DAS_Tool v1.1.1, pullseq v1.0.2, Ruby v3.0.1 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/Ruby/Ruby-3.0.1-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+exts_default_options = {
+    'source_urls': ['https://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
+# !! order of packages is important !!
+# some packages have dependencies with minimum and maximum version requirements
+# each version is picked as high as possible to fullfill all requirements
+# packages updated on 2021-07-02
+exts_list = [
```
Comment (smoors) [inline easybuild/easyconfigs/r/Ruby/Ruby-3.0.1-GCCcore-11.2.0.eb:26]: @ItIsI-Orient all these extensions should ideally be updated to their most recent version, if possible.

**Review by smoors — DISMISSED** [review]

File: easybuild/easyconfigs/r/Ruby/Ruby-3.0.1-GCCcore-11.2.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/r/Ruby/Ruby-3.0.1-GCCcore-11.2.0.eb:26]: I realize this is a lot of work, maybe you can ask in Slack or the mailing list if someone has a script for that?

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3589.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/ed1c0b5707b869933516adf5ccfba577 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in total)
node2620.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/aa22daa023cb324cde963632437e4fd6 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**